eng/common/scripts/Test-SampleMetadata.ps1

```diff
@@ -104,6 +104,7 @@ begin {
         "azure-active-directory-b2c",
         "azure-active-directory-domain",
         "azure-advisor",
+        "azure-ai-contentsafety",
```

Comment:
I don't see this one in https://review.learn.microsoft.com/en-us/help/platform/metadata-taxonomies?branch=main#product so I don't expect the docs system to understand it yet.

Reply:
We have name azure-ai-content-safety
But is should be updated to azure-ai-contentsafety
@catalinaperalta